### ISSUE TITLE
Tweet's format personalization

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#43]
```
Maybe a new panel at the Preferences menu to let the user personalize the 
tweets appearance like font, size, color

What version of the product are you using? On what operating system?
Pino 0.1.3 on Ubuntu 9.10
```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 13 Jan 2010 at 9:22

**Comment by GoogleCodeExporter** [comment]
```
Maybe it will be a font and a color for unreaded tweets. I don't want to do a 
full 
customizing.
```

Original comment by `tro...@gmail.com` on 14 Jan 2010 at 10:54
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
oh ok, that's fine. I would really appreciate the size option to be included, 
because 
a smaller one would let the user to see more tweets at a time :D

However this app's great. I really wanted to put my hands on a non Adobe Air 
alternative. Simple, light, just what i wished! Keep on this!
```

Original comment by `mich.ba...@gmail.com` on 14 Jan 2010 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
+1 for smaller fonts. Think 'netbooks' here, :)
```

Original comment by `diesph...@gmail.com` on 1 Feb 2010 at 11:10

**Comment by GoogleCodeExporter** [comment]
```
If you know some CSS you can change the font by editing the file
/usr/share/pino/templates/main.tpl. But i would appreciate a option, too.
```

Original comment by `neunerch...@gmail.com` on 2 Feb 2010 at 7:54

**Comment by GoogleCodeExporter** [comment]
```
thanks for the tip!
```

Original comment by `mich.ba...@gmail.com` on 3 Feb 2010 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
Closed by 00f2664b8e revision.
```

Original comment by `tro...@gmail.com` on 9 Feb 2010 at 8:43
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Yeah!!
```

Original comment by `mich.ba...@gmail.com` on 10 Feb 2010 at 2:20
